cfme/infrastructure/virtual_machines.py

```diff
@@ -215,8 +215,9 @@ class ProviderTemplatesOnlyAllView(TemplatesOnlyAllView):
 
     @property
     def is_displayed(self):
-        text = 'Miq' if self.browser.product_version < "5.9" else 'VM'
-        msg = '{} (All {} Templates)'.format(self.context['object'].name, text)
+        text = 'Miq' if self.browser.product_version < '5.9' else 'VM'
```

Comment:
Required:
Remove this as we are not supporting `5.8` against master branch. You can commit this against `5.8 break branch`